gnd/src/main/java/com/google/android/gnd/ui/map/MapFragment.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2018 Google LLC
+ * Copyright 2021 Google LLC
```

Comment:
We're not supposed to update this. You could change it to `2018-2021`, though.

Reply:
`MapProviderModule` was deleted and `MapFragment` was added. For some reason, GitHub thinks thinks `MapProviderModule` is renamed to `MapFragment` and then updated. Maybe a bug with github?

Reply:
Ahh ok, then yeah please leave as is! Though I do wonder what `git log` and `git show` think about it. The underlying VCS might be what thinks it was renamed. Not sure what to do re: copyrights in the case of a rename.